src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,123 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75F;
+    private int size;
+    private MyNode<K,V>[] bucketArray;
+
+    private class MyNode<K,V> {
```

Comment:
![image](https://user-images.githubusercontent.com/20326858/201346904-ffbc1520-7fdf-45b8-bb50-da991dc91fbb.png)

Reply:
private inner class should be after public methods